setup.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-from __future__ import absolute_import
+from __future__ import absolute_import, print_function
```

Comment:
I don't think they have to.  I was just attempting to mimic the style of the next line, which does two imports in one shot.